src/main/mpack/common-services/SOLR/5.5.5/metainfo.xml

```diff
@@ -32,7 +32,10 @@
                     <osFamily>any</osFamily>
                     <packages>
                         <package>
-                            <name>lucidworks-hdpsearch</name>
+                          <name>lsof</name>
```

Comment:
I'm pretty sure that `lsof` is a sufficient package name on the systems we support.

I've verified this in SLES 12.3, in Ubuntu 14.04 and 16.04, in Centos 6 and 7, in RedHat 7.5.

There's a few particular OS/version combinations I haven't tried specifically, but all of those also look good, based on some poking around online.

- https://packages.debian.org/search?keywords=lsof
- https://software.opensuse.org/package/lsof
- https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/6.3_technical_notes/lsof
- https://packages.ubuntu.com/search?keywords=lsof